behavior_metrics/brains/brains_handler.py

```diff
@@ -53,7 +55,7 @@ def load_brain(self, path, model=None):
             else:
                 if model:
                     self.active_brain = Brain(self.sensors, self.actuators, model=model, handler=self, config=self.config)
-                elif hasattr(self, 'model'):
+                elif self.model is not None:

```

Comment:
This change causes a merge issue. The code here should be as in the noetic-devel branch: 

```
def load_brain(self, path, model=None):

        path_split = path.split("/")
        robot_type = path_split[-2]
        module_name = path_split[-1][:-3]  # removing .py extension
        import_name = 'brains.' + robot_type + '.' + module_name

        if robot_type == 'CARLA':
            module = importlib.import_module(import_name)
            Brain = getattr(module, 'Brain')
            if self.model:
                self.active_brain = Brain(self.sensors, self.actuators, handler=self, model=self.model, config=self.config)
            else:
                self.active_brain = Brain(self.sensors, self.actuators, handler=self, config=self.config)
        else:
            if import_name in sys.modules:  # for reloading sake
                del sys.modules[import_name]
            module = importlib.import_module(import_name)
            Brain = getattr(module, 'Brain')
            if robot_type == 'drone':
                self.active_brain = Brain(handler=self, config=self.config)
            else:
                if model:
                    self.active_brain = Brain(self.sensors, self.actuators, model=model, handler=self, config=self.config)
                else:
                    self.active_brain = Brain(self.sensors, self.actuators, handler=self, config=self.config)

```